Makefile

```diff
@@ -43,9 +43,11 @@ postgres:
 	@echo 'export POSTGRESQL_CONNECTION="host=localhost port=5432 user=postgres dbname=postgres password=password123"'
 
 
+LINT_FIX ?= --fix
```

Comment:
Maybe `LINT_ARGS` ? Since you can set any arguments with this. I'll make that change.